### ISSUE TITLE
Sign up for a user

### ISSUE THREAD

**Issue by jcooky** [tymsai/marmoym-api#79]

**Comment by gimochi** [comment]
{
  "username": "usernametest123",
  "password": "123qwe",
  "email": "minsubkim924@gmail.com"
}

**Comment by gimochi** [comment]
```
{
  "username": "gimochi",
  "password":  "123qwe",
   "email": "minsubkim94@gmail.com"
}
```

**Comment by gimochi** [comment]
Fixed by #96
